File: Lista 07 - Strings/B.c
#include <stdio.h>
#include <string.h>

int main(){
    char str1[1001], str2[1001];
    scanf("%s %s", str1, str2);

    int cord_str1 = -1, cord_str2 = -1; 
    for(int i = 0; str1[i] != '\0'; i++){
        for(int j = 0; str2[j] != '\0'; j++){
            if(str1[i] == str2[j]){
                cord_str1 = i + 1;
                cord_str2 = j + 1;
            }
        }
    }

    printf("%d %d\n", cord_str1, cord_str2);

    return 0;
}

//🗣 esse código faz exatamente a mesma coisa que o B.py, mas em python ele não funciona (TLE) e em C funciona 🔥

File: Lista 07 - Strings/B.py
str1 = input()
str2 = input()

cord_str1 = -1
cord_str2 = -1
for i_idx, i in enumerate(str1):
    for j_idx, j in enumerate(str2):
        if i == j:
            cord_str1 = i_idx + 1
            cord_str2 = j_idx + 1

print(cord_str1, cord_str2)